.github/workflows/release.yml

```diff
@@ -58,30 +57,24 @@ jobs:
         shell: bash
 
       - name: Install dependencies
-        working-directory: ./vscode
         run: npm ci
 
-      # Run tests on Linux
+      # Run tests on different OSes
       - name: Run tests (Linux)
         if: matrix.arch == 'linux'
         working-directory: ./vscode
```

Comment:
should we remove also this one? if this is a special case please add a comment

Reply:
Yeah it does look like this working-directory line should be removed.